<launch>

  <arg name="bag_file" default="$(find aracati2017)/bags/ARACATI_2017_8bits_full.bag"/>
  <arg name="image_path" default="$(find aracati2017)/maps/2021-12.yaml"/>

  <param name="use_sim_time" value="true" />

  <node pkg="aracati2017" name="odom" type="odom"
        respawn="false" output="screen">
    <remap from="/pose" to="/pose_gt"/>
    <param name="odom_frame_id" value="map" />
    <param name="base_frame_id" value="son" />
  </node>

  <node pkg="aracati2017" name="aerial_image"
        type="aerial_image"
        respawn="false" output="screen">
        <remap from="/odom" to="/odom_pose"/>
        <remap from="/pose_gt" to="/pose_gt"/>
    <param name="aerial_image_path" value="$(arg image_path)" />
    <param name="son_fov_openning" value="130.0" />
    <param name="son_min_range" value="0.4" />
    <param name="son_max_range" value="50.0" />
  </node>

  <node pkg="rqt_gui" name="son_config" type="rqt_gui"
        respawn="false" args="--perspective-file $(find aracati2017)/cfg/view.perspective">
  </node>

  <node pkg="rosbag" name="rosbag_play" type="play"
        respawn="false" output="screen"
        args="$(arg bag_file) --clock -r 4.0 -q">
  </node>

</launch>
